### PR TITLE
Configurable 3d sensor

### PR DESCRIPTION
**Pull request by jihoonl**
This PR allows to configure TURTLEBOT_3D_SENSOR for turtlebot.
- kinect
- asus_xtion_pro : RGB sensor is located at center. (RGB sensor location : -0.087, **-0.0125**, 0.287)
- asus_xtion_pro_center : Device is located at center. (RGB sensor location : -0.087, **0.0205**, 0.287)

turtlebot

### CODE REVIEW

**Comment by jihoonl** [comment]
Don't merge until we discuss about new xtion pro position.

**Comment by jihoonl** [comment]
Mesh and urdf need to be updated. I will get back with updates.

**Comment by LucidOne** [comment]
You need any files from me?

**Comment by jihoonl** [comment]
So far no. I will let you know if anything is need.

**Comment by stonier** [comment]
@jihoonl what do you think about switching the identifier? Since we're prototyping with the centred asus as the default, make them `asus_xtion_pro` and `asus_xtion_pro_offset`?

**Comment by jihoonl** [comment]
Do You mean `asus_xtion_pro` for new version and `asus_xtion_pro_offset` for old version?
Wouldn't it make confusion to old users who have already been using asus with old configuration?

**Comment by stonier** [comment]
We are already changing the defaults (kinect to asus) and have agreed the best thing is to make it simple for new users. To look after the old users we make sure we are very clear in a migration guide. I do not wish to go half one way, then half the other - that is confusing.

**Comment by jihoonl** [comment]
Now
- kinect
- asus_xtion_pro : new location
- asus_xtion_pro_offset : old location.

**Comment by stonier** [comment]
Awesome...

@tfoote I'm pushing this in so we can move on getting the notebook installation ready for  manufacturing (running short on time there). I'll update the turtlebot-distributors list with drawing and specs for mounting to get feedback. Should all be ok, but if we do get any 'must change' feedback, we can revisit this then.

**Comment by stonier** [comment]
@jihoonl you said you need to update the mesh yet...would you like to open a new pull request for that when ready.

File: turtlebot_description/urdf/turtlebot_properties.urdf.xacro [hunk 1]
```diff
@@ -23,7 +23,7 @@
      (currently in development and awaiting feedback)
    -->
 	<property name="cam_px" value="-0.087" />
-	<property name="cam_py" value="-0.0125" />
+<!--	<property name="cam_py" value="-0.0125" />-->
```
Comment (stonier) [inline turtlebot_description/urdf/turtlebot_properties.urdf.xacro:26]: Might want to add some explanation here, or link to a pull request. It's a bit obscure for users and/or future devs having it just 'commented'.

**Comment by jihoonl** [comment]
I have added comments with direct commit. And new mesh has already been applied with this PR.

https://github.com/turtlebot/turtlebot/commit/b1c2c917cc8b7a60bf729a432ce82e274017af5d

**Comment by 130s** [comment]
I don't know what's the correct and optimal instruction for this change that affects Turtlebot2 with Kinect, but I made [this change on wiki](http://wiki.ros.org/action/info/turtlebot/Tutorials/indigo/3D%20Visualisation?action=diff&rev2=12&rev1=11).

**Comment by jihoonl** [comment]
@130s 

The kinect configuration should have done in [turtlebot bringup](http://wiki.ros.org/turtlebot_bringup/Tutorials/indigo/TurtleBot%20Bringup) ROS Environment configuration and [Kinect setup](http://wiki.ros.org/turtlebot/Tutorials/indigo/Kinect%20Setup).

However, the guide looks still necessary to guide people only uses 3d sensors. I have updated the wiki to configure robot to use Kinect in proper way. [Change](http://wiki.ros.org/action/info/turtlebot/Tutorials/indigo/3D%20Visualisation?action=diff&rev2=13&rev1=12)

**Comment by 130s** [comment]
Thanks @jihoonl!

**Pull request merged**